met-api/src/met_api/resources/comment.py

```diff
@@ -54,42 +54,6 @@ def get(comment_id):
 
 
 @cors_preflight('GET, OPTIONS')
-@API.route('/submission/<submission_id>')
```

Comment:
Is this resource still used for any operation anyway?

Reply:
yes! public view and text listing where all the comments get displayed